packages/core/src/components/sections/ProductDetails/Overrides.tsx

```diff
@@ -0,0 +1,11 @@
+import { Price, DiscountBadge } from '@faststore/ui'
+
+import ProductDetailsCustomizations from 'src/customizations/components/overrides/ProductDetails'
+
+const Components = {
+    Price,
+    DiscountBadge,
+    ...ProductDetailsCustomizations.components
```

Comment:
here must be listed all components of `faststore/ui` right?

Reply:
Only the ones we will allow to be overwritten